src/views/transaction-details/transaction-details.styles.ts

```diff
@@ -0,0 +1,116 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+import { TransactionCardProps } from "src/views/activity/components/transaction-card/transaction-card.view";
+
+interface StylesProps {
+  status: TransactionCardProps["status"];
+}
+
+const useTransactionDetailsStyles = createUseStyles((theme: Theme) => ({
+  card: {
+    padding: theme.spacing(3),
+  },
+  dot: {
+    width: theme.spacing(0.75),
+    height: theme.spacing(0.75),
```

Comment:
Using theme spacing to define sizes will produce unexpected size changes when one decides to change the spacing.

Reply:
I agree with this

Reply:
Fixed